protobuf/transform.go

```diff
@@ -167,6 +182,14 @@ func (t *Transformer) transformStruct(pkg *Package, s *scanner.Struct) *Message
 	return msg
 }
 
+func defaultOptionsForScannedMessage(s *scanner.Struct) Options {
+	opts := make(Options)
+
+	opts["(gogoproto.drop_type_declaration)"] = NewLiteralValue("true")
+
+	return opts
```

Comment:
You can't abstract it because it requires different concrete types. And also the options are different.